home.html

```diff
@@ -14,7 +14,7 @@
   <div class="mission">
     <div class="container">
       <p>
-        People are sharing more and more photos and videos every day. Mobile networks
+        Men are sharing more and more photos and videos every day. Mobile networks
```

Comment:
haha, no that was just me trolling around to see if a live rebuilding worked locally. Fixing!